app/slashcommands/command_remote.go

```diff
@@ -129,7 +129,7 @@ func (rp *RemoteProvider) doInvite(a *app.App, args *model.CommandArgs, margs ma
 
 	rcSaved, appErr := a.AddRemoteCluster(rc)
 	if appErr != nil {
-		return responsef("Could not add remote cluster: %v", appErr)
```

Comment:
I would suggest to use translation templates to pass variables, and avoid passing the formatting characters as part of the translation string itself. It looks a bit weird for a translator to see `%v` in the middle of a string. It leaks abstractions.

Something like this:
`T("tx_code", map[string]interface{}{"appErr": appErr.Error()})`

Reply:
I see, seems fair.